src/ucp/api/ucp.h

```diff
@@ -1439,6 +1458,24 @@ typedef struct {
      * This field must be initialized by the caller.
      */
     size_t            message_size;
+
+    /**
+     * Rkey of the remote memory region to be used for the operation.
+     * This field must be initialized by the caller.
```

Comment:
i'd remove this phrase for all fields and maybe mention it once for the whole struct. For this API we have separate params and attrs structures, so it is supposed that params is fully initialized by the caller